crates/genesis/src/lib.rs

```diff
@@ -226,7 +230,7 @@ pub struct GenesisAccount {
         skip_serializing_if = "Option::is_none",
         deserialize_with = "deserialize_storage_map"
     )]
-    pub storage: Option<HashMap<B256, B256>>,
+    pub storage: Option<BTreeMap<B256, B256>>,
```

Comment:
I actually prefer the BtreeMap change here so serialize/deserialize is consistent